website/src/components/Header.tsx

```diff
@@ -35,6 +36,31 @@ function MobileNavLink({ children, ...props }) {
   );
 }
 
+function AccountButton() {
+  const { data: session } = useSession();
+  if (session) {
+    return (
+      <button
+        type="button"
+        onClick={() => signOut()}
+        className="inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
+      >
+        Log out
+      </button>
+    );
+  }
+  return (
+    <Link href="/auth/signin" aria-label="Home" className="flex items-center">
+      <button
+        type="button"
+        className="inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
+      >
+        Log in
+      </button>
```

Comment:
button looks exactly the same as the button above, could this be a component?

Reply:
I agree, this was originally from @k-nearest-neighbor.  I'm leaving css and component cleanup for later changes.